style.css

```diff
@@ -6,377 +7,730 @@
-  background: #f7f7f9;
-}
-
-header {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  height: 48px;
-}
-
-.bg-white {
-  background-color: #fff;
-}
-
 .logo {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 10px 12px;
-  margin-left: 12px;
-  margin-top: 4px;
-}
-
-.logo a {
-  text-decoration: none;
+  font-style: normal;
   font-weight: 700;
   font-size: 18px;
   line-height: 20px;
   color: #6070ff;
+  list-style: none;
+  width: 100px;
+  display: inline-block;
 }
 
-nav {
-  padding: 8px;
+a {
+  text-decoration: inherit;
+  color: inherit;
+  cursor: auto;
 }
 
-.menu-icon {
-  width: 24px;
-  height: 24px;
-  margin-right: 12px;
-  cursor: pointer;
+body {
+  font-family: "Poppins", sans-serif;
+  width: 100%;
+  height: 100%;
+  overflow-x: hidden;
+  background-color: #f7f7f7;
 }
 
-section {
-  display: grid;
-  grid-template-columns: 1fr;
-  padding: 114px 24px;
-  gap: 12px;
+nav {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  width: 100%;
+  justify-content: space-between;
+  background-color: white;
+  padding: 10px 20px;
 }
 
 ul {
+  display: flex;
+  flex-direction: row;
+  gap: 35px;
   list-style: none;
+  margin-right: 686px;
 }
 
 .headline {
-  background-image: url("./images/header-shapes-mobile.png");
+  background-image: url(./portfolio/Background.png);
+  width: 100vw;
+  height: 100vh;
+  background-color: #fff;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-bottom: 20px;
+  border-bottom-left-radius: 10%;
+}
+
+.title {
   display: flex;
   flex-direction: column;
-  align-items: flex-start;
-  padding: 114px 24px;
-  gap: 12px;
-  border-bottom-left-radius: 60px;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  top: 20%;
+  width: 100%;
 }
 
-.title {
+h1 {
+  width: 920px;
+  height: 122px;
+  color: #091e42;
+  font-style: normal;
   font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  color: #172b4d;
+  font-size: 48px;
+  line-height: 60px;
+  letter-spacing: 0.37px;
 }
 
 .details {
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
+  width: 920px;
+  height: 98px;
   color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  overflow-wrap: break-word;
 }
 
-.contact-link {
-  text-decoration: none;
+.links-text {
+  font-style: normal;
   font-weight: 500;
-  font-size: 16px;
+  font-size: 18px;
   line-height: 24px;
   color: #7f8cff;
+  margin-top: 7px;
+  margin-bottom: 20px;
+  margin-right: 807px;
 }
 
-.social-media-links {
-  margin-top: 17px;
-  display: flex;
-  gap: 20px;
-  text-decoration: none;
-}
-
-.works {
+.project_section {
+  background-color: rgb(229, 229, 229);
+  justify-content: center;
+  padding: 0 24px;
   display: grid;
-  flex-direction: column;
-  align-items: center;
-  padding: 114px 24px;
-  gap: 88px;
+  grid-template-rows: auto;
+}
+
+@media (max-width: 600px) {
```

Comment:
[OPTIONAL] Kindly note that the width of the mobile version is between 375px and 768px and not up to 600px. You will always have issues since you are starting the desktop version at `min-width 768px`. Please check and correct this anomaly in order to have a responsive page. :wink: